### PR TITLE
Add Grafana yaml example

### PR DESCRIPTION
**Pull request by rgrandl**
Example to start Grafana and add the Jaeger/Prometheus as data sources to Grafana.

We also provide an example dashboard that enables the user to visualize the Service Weaver generated metrics. This can be used as a starting point to create more advanced dashboards for Service Weaver applications.

### CODE REVIEW

File: examples/telemetry/grafana.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            name: grafana-config
+            subPath: grafana.yaml
+      volumes:
+      - name: grafana-config
+        configMap:
+          name: grafana-config
+
+---
+
+# Grafana Service
```
Comment (mwhittaker) [inline examples/telemetry/grafana.yaml:46]: I think Kubernetes encourages you to put a Service before the corresponding Deployment, so I would move this to the top.

**Review by mwhittaker — APPROVED** [review]
Awesome :fire:

File: examples/telemetry/grafana.yaml (hunk 1 above)
Comment (rgrandl) [reply examples/telemetry/grafana.yaml]: Done.

**Review by rgrandl — COMMENTED** [review]
Thanks Michael!

**Pull request merged**